config/crd/bases/ssp.kubevirt.io_ssps.yaml

```diff
@@ -19,8 +19,6 @@ spec:
     - jsonPath: .status.phase
       name: Phase
       type: string
-    deprecated: true
-    deprecationWarning: ssp.kubevirt.io/v1beta1 ssp is deprecated
     name: v1beta1
```

Comment:
Why are we dropping the deprecation?

Reply:
Because I kept the storage version `v1beta1`. We can deprecate it in a later PR.

Reply:
What's the reasoning behind keeping the storage version `v1beta1`? Moving it to `v1beta2` shouldn't have any impact for us as https://github.com/openshift/cluster-kube-storage-version-migrator-operator should always be present in our environment?

Reply:
Ok, I didn't know about it. I will change it back.